bin/iam-token.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# the iam-token.sh scripts generate new IAM token
+# it relies on the following environment variables
+# IAM_CLIENT_ID client id value
+# IAM_CLIENT_SECRET client secret value
+# IAM_TOKEN output file name to store obtained IAM token
+# All steps to obtain client credentials can be found:
+# https://github.com/dmwm/WMCore/pull/11093#issuecomment-1098131010
+
+# check if curl exist on a system
+if ! command -v curl &> /dev/null
+then
+    echo "curl could not be found, please install it on your system"
+    exit
+fi
+# check if jq exist on a system
+if ! command -v jq &> /dev/null
```

Comment:
I checked lxplus and one CERN agent and I confirm this JSON tool exists. Still have to check the FNAL node.
Nonetheless, can you please update the initial PR description mentioning that this `jq` package is required in the node running a token-based WMCore service?